hackage-security/hackage-security.cabal

```diff
@@ -141,8 +141,8 @@ library
                        cryptohash-sha256 >= 0.11    && < 0.12,
                        -- 0.4.2 introduces TarIndex, 0.4.4 introduces more
                        -- functionality, 0.5.0 changes type of serialise
-                       tar               >= 0.5     && < 0.6,
-                       template-haskell  >= 2.7     && < 2.20,
+                       tar               >= 0.5     && < 0.7,
```

Comment:
@TristanCacqueray : There seems to be no `tar-0.6.*`: https://hackage.haskell.org/package/tar

Reply:
Oh oops, I got it from the project HEAD to get it to compile with ghc-9.6 ( https://github.com/haskell/tar/blob/master/tar.cabal )